### PR TITLE
[CF-198] Write stdout to log

### PR DESCRIPTION
**Pull request by SVilgelm**
Forward the stdout messages into log with INFO level

### CODE REVIEW

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by roman-verchikov** [comment]
+1

File: configs/logging_config.yaml [hunk 1]
```diff
@@ -53,7 +53,7 @@ handlers:
     date_format: '%F-%H-%M-%S'
     formatter: debug_colored
     level: NOTSET
-    maxBytes: 10485760
+    maxBytes: 1073741824
```
Comment (roman-verchikov) [inline configs/logging_config.yaml]: Maybe it supports Mbytes or something? 10-digit number looks really odd

Comment (SVilgelm) [inline configs/logging_config.yaml]: Done

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by SVilgelm** [comment]
@roman-verchikov, May we merge this? Or we need any comments from @archyufa?

**Comment by roman-verchikov** [comment]
can be merged

**Pull request merged**